### ISSUE TITLE
Uncaught TypeError: J.j(...).gcz is not a function  IR Tab with Disassembly

### ISSUE THREAD

**Issue by rawbin-** [mraleph/irhydra#58]
```
var i,len,testData = [],testCount = 9999,repeatCount = 999;

function doInit(testData){
  for(var i = testCount; i--;){
  	testData[i] = i;
  }
}

function doRepeat(operation,data){
  for(var i = 0; i < repeatCount;i++){
    operation(data);
  }
}
function origin(data){
  var ret = [];
  for(var i = 0; i < data.length; i++){
    ret[i] = data[i] * data[i];
  }
}

function optLen(data){
  var ret = [];
  for(var i = 0,len = data.length; i < len; i++){
    ret[i] = data[i] * data[i];
  }
}

function optIdx(data){
  var ret = [],tmpData;
  for(var i = 0,len = data.length; i < len; i++){
    tmpData = data[i];
    ret[i] = tmpData * tmpData;
  }
}

function optCmp(data){
  var ret = [];
  for(var i = data.length; i-- > 0;){
    ret[i] = data[i] * data[i];
  }
}

doInit(testData);

console.time("origin")
doRepeat(origin,testData);
console.timeEnd('origin')

console.time('optmize len');
doRepeat(optLen,testData)
console.timeEnd('optmize len');

console.time('optmize index');
doRepeat(optIdx,testData)
console.timeEnd('optmize index');

console.time('optmize compare');
doRepeat(optCmp,testData)
console.timeEnd('optmize compare');
```

**Comment by rawbin-** [comment]
With Node 7.9.0 on Mac
```
node --trace-hydrogen --trace-phase=Z --trace-deopt --code-comments --hydrogen-track-positions --redirect-code-traces --redirect-code-traces-to=code.asm --print-opt-code perf-opt.js
```

**Comment by rawbin-** [comment]
![irhydra](https://cloud.githubusercontent.com/assets/8169783/25841880/6fa823fa-34d4-11e7-83fa-3a32fa4fe219.png)

**Comment by mraleph** [comment]
Checking it out.

**Comment by mraleph** [comment]
@rawbin- fixed by 0b3525398592008b17551f41b0dd7f745a1b1031 and redeployed. 

Please try again.
